moto/cloudwatch/models.py

```diff
@@ -1,12 +1,13 @@
-import json
 
+import json
+from moto.core.utils import iso_8601_datetime_with_milliseconds
 from moto.core import BaseBackend, BaseModel
 from moto.core.exceptions import RESTError
 import boto.ec2.cloudwatch
-import datetime
-
+from datetime import datetime, timedelta
+from dateutil.tz import tzutc
 from .utils import make_arn_for_dashboard
-
+import statistics
```

Comment:
Is this available on Python2? and if not, its only used for mean, couldnt that be replaced with sum() / len()

Reply:
new in 3.4 :(  I'll update it

Reply:
Yeah I know how you feel, shame moto can't be py35+ only